app/controllers/status_controller.rb

```diff
@@ -15,12 +15,24 @@ def index
   # This is useful for troubleshooting authentication with access tokens and
   # configuring proxies or load balancers.
   def whoami
-    render json: { 
+    audit_success
+
+    render json: {
       client_ip: request.ip,
       user_agent: request.user_agent,
       account: token_user.account,
       username: token_user.login,
       token_issued_at: Time.at(token_user.token.claims["iat"])
     }
   end
+
+  def audit_success
+    @role = ::Role.by_login(token_user.login, account: token_user.account)
```

Comment:
Do we have to define role as class member (with @)? Maybe it can be a simple variable?